src/libtiled/mapreader.cpp

```diff
@@ -774,6 +800,16 @@ void MapReaderPrivate::readTilesetWangSets(Tileset &tileset)
                     wangTile.setFlippedVertically(fV);
                     wangTile.setFlippedAntiDiagonally(fA);
 
+                    if (!fInherit) {
+                        const bool canH = tileAtts.value(QLatin1String("can_hflip")).toInt();
+                        const bool canV = tileAtts.value(QLatin1String("can_vflip")).toInt();
+                        const bool canA = tileAtts.value(QLatin1String("can_dflip")).toInt();
```

Comment:
I would suggest we reuse the `hflip`, `vflip` and `dflip` attributes here, since they were never used before. I mean the user never had a way to set those to `true` anyway.

Reply:
Oh, true, I was overly cautious.

Reply:
On the other hand it feels strange to reuse a(n unused) short name for a rare property.